src/server/index.jsx

```diff
@@ -283,7 +285,12 @@ server.get(
         });
       }
 
-      logger.info(ROUTING_INFORMATION, {
+      let routingInfoLogger = logger.debug;
+      if (status !== OK) {
+        routingInfoLogger = logger.error;
```

Comment:
Question: should this be `warning` or `error`?

Reply:
Maybe warning as a 404 is not necessarily an error, it is a valid response to requesting an non-existent resource? You could consider 500 an error but I'm not sure the distinction is important enough 🤷

Reply:
Let's make >=400 and <499 a warning, and >= 500 an error. Because technically redirects and other statuses in in the 200 - 300 range are also OK :)